cmd/asset-syncer/server/utils.go

```diff
@@ -518,7 +518,7 @@ func pullAndExtract(repoURL *url.URL, appName, tag string, puller helmutils.Char
 
 func chartImportWorker(repoURL *url.URL, r *OCIRegistry, chartJobs <-chan pullChartJob, resultChan chan pullChartResult) {
 	for j := range chartJobs {
-		log.WithFields(log.Fields{"name": j.AppName, "tag": j.Tag}).Debug("pulling chart")
+		log.V(4).Infof("pulling chart, name=%s, tag=%s", j.AppName, j.Tag)
```

Comment:
Why not use klog's support for structured logging here and above, with something like:
```suggestion
		log.V(4).InfoS("pulling chart", "name", j.AppName, "tag", j.Tag)
```

or even

```suggestion
		log.V(4).InfoS("pulling chart", "chartJob", j)
```

Better eg showing the difference: https://go.dev/play/p/EW5ZQh7MDXI

Or maybe you want to do that separately so it's consistent across the codebase (personally I'd think it OK to do it as we update, like here, but up to you).

Reply:
Thanks for the suggestion and the playground example. I haven't given it a try yet. Really interesting.

As per https://github.com/kubeapps/kubeapps/issues/3848#issuecomment-1019959653, I already have planned to do so in a separate PR once this stack of PRs became merged. I've added a card to our board not to forget about it.